build

```diff
@@ -83,7 +83,7 @@ set -o errexit
 
 # Check for dependencies
 for dep in docker make mmv; do
-    which ${dep} || { echo "The command '${dep}' is required."; exit 1; }
+    command -v ${dep} &>/dev/null || { echo "The command '${dep}' is required."; exit 1; }
```

Comment:
should be `command -v "${dep}"` in case `dep` contains a space. I know it most likely wouldn't, so it's a nitpicking... but still %)